SpatialGDK/Source/SpatialGDKEditor/Private/SchemaGenerator/Utils/DataTypeUtilities.cpp

```diff
@@ -27,6 +27,10 @@ FString GetEnumDataType(const UEnumProperty* EnumProperty)
 
 FString UnrealNameToSchemaName(const FString& UnrealName)
 {
+	if (UnrealName.IsValidIndex(0) && FChar::IsDigit(UnrealName[0]))
+	{
+		return "ZZ" + AlphanumericSanitization(UnrealName);
```

Comment:
Any reason for ZZ?

Reply:
I wanted to keep it short and, given the frequency of the character Z in English, it's highly unlikely that someone would choose ZZ as a prefix themselves, making it very obvious that this class was internally renamed. I'm open to other suggestions, though.